lib/api/include/irods/get_resource_info_for_operation.h

```diff
@@ -9,56 +10,67 @@ struct RcComm;
 ///
 /// \param[in] _comm An rcComm_t connection handle to the server.
 /// \param[in] _dataObjInp \parblock A DataObjInp describing the data object operation.
 /// - \p objPath: The logical path of the target data object.
 /// - \p condInput: A list of options that influence the results of the operation.
-///     - \p REPL_NUM_KW  - The replica number of the copy to upload.
-///     - \p GET_RESOURCE_INFO_FOR_OPERATION_KW - The operation of interest. The following values are supported:
+///     - \p REPL_NUM_KW: The replica number of an existing replica.
+///     - \p GET_RESOURCE_INFO_FOR_OPERATION_KW: The operation of interest. The following values are supported:
 ///         - "CREATE"
 ///         - "WRITE"
 ///         - "OPEN"
 ///         - "UNLINK"
-///     - \p RESC_NAME_KW - Resource name hint.  Given the name of a root resource, influences voting to select the appropriate replica.
-///     - \p RESC_HIER_STR_KW -  Resource hierarchy hint.  Given a hierarchy string, specifies the desired replica.
-/// \endparblock
-/// \param[out] _out_info \parblock a JSON string with keys "host" and "resource_hierarchy", identifying target server and supplying a
-///                                 resource hierarchy string denoting a specific data object replica for the logical path given as input.
+///     - \p RESC_NAME_KW: Resource name hint. Given the name of a root resource, influences voting to select the
+///                        appropriate replica.
+///     - \p RESC_HIER_STR_KW: Resource hierarchy hint. Given a hierarchy string, specifies the desired replica.
 /// \endparblock
+/// \param[out] _out_info \parblock a JSON string with keys "host" and "resource_hierarchy", identifying target server
+///                       and supplying a resource hierarchy string denoting a specific data object replica for the
+///                       logical path given as input.
 ///
-/// \b Example _out_info:
-///
+/// On success, \p _out_info will have the following structure:
 /// \code{.js}
 /// {
-///     "host": <string>,
-///     "resource_hierarchy": <string>
+///     "host": string,
+///     "resource_hierarchy": string
 /// }
 /// \endcode
+/// \endparblock
 ///
 /// \b Example usage:
 ///
 /// \code{.cpp}
+/// RcComm* comm = // Our iRODS connection handle.
+///
 /// DataObjInp inp;
 /// memset(&inp, 0, sizeof(DataObjInp));
+///
 /// strncpy(inp.objPath, "/tempZone/home/rods/data_object", MAX_NAME_LEN);
-/// char* char_buffer = 0;
-/// addKeyVal(&input.condInput, "GET_RESOURCE_INFO_OP_TYPE_KW", "CREATE");
-/// addKeyVal(&input.condInput, "RESC_NAME_KW", "my_root_resource");
-/// const int ec = rc_get_resource_info_for_operation(&comm, &inp, &char_buffer);
+/// char* char_buffer = NULL;
+///
+/// addKeyVal(&inp.condInput, "GET_RESOURCE_INFO_OP_TYPE_KW", "CREATE");
+/// addKeyVal(&inp.condInput, "RESC_NAME_KW", "my_root_resource");
+///
+/// const int ec = rs_get_resource_info_for_operation(comm, &inp, &char_buffer);
```

Comment:
Add more words explaining how to interpret these.

_Apply to other header file._

Reply:
Done.